hack/machineset.sh

```diff
@@ -414,28 +419,24 @@ platform="$(oc get infrastructure cluster -ojsonpath={.spec.platformSpec.type})"
 # Gets the Infrastructure Id for the cluster like `pmahajan-azure-68p9l-gv45m`
 infraID="$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)"
 
-# Determines the region based on existing MachinesSets like `us-east-1` for aws or `centralus` for azure
-region="$(oc get machines -n openshift-machine-api | grep -w "Running" | awk '{print $4}' | head -1)"
-
-# Determines the availability zone based on existing MachinesSets like `us-east-1a` for aws or `2` for azure
-az="$(oc get machines -n openshift-machine-api | grep -w "Running" | awk '{print $5}' | head -1)"
+linuxWorkerSpec=$(oc get machines -n openshift-machine-api -l machine.openshift.io/cluster-api-machine-role=worker -ojsonpath={.items[0].spec})
```

Comment:
looks like we are losing the `"Running"` filter, should this continue looking for linuxWorkerSpec of Running machines?

Reply:
Is that an edge case that needs to be handled by this script? Seems like it would only be an issue if you tried to run this while  the cluster was upgrading. In order to handle that scenario we would have to loop through each machine and inspect the status.